README.md

```diff
@@ -116,25 +165,75 @@ j2 = Rotations.jacobian(q, p) # How does the rotated point q*p change w.r.t. the
+    
+6. **Rodrigues Parameters** `RodriguesParam{T}`
+    A 3-parameter representation of 3D rotations that has a singularity at 180 degrees. They can be interpreted as a projection of the unit quaternion onto the plane tangent to the quaternion identity. They are computationally efficient and do not have a sign ambiguity. 
+    
+7. **Modified Rodrigues Parameter** `MRP{T}`
+    Equivalent to `SPQuat{T}`. Are frequently used in aerospace applications since they are a 3-parameter representation whose singularity happens at 360 degrees. In practice, the singularity can be avoided with some switching logic between one of two equivalent MRPs (obtained by projecting the negated quaternion). 
 
-6. **Cardinal axis rotations** `RotX{T}`, `RotY{T}`, `RotZ{T}`
+8. **Cardinal axis rotations** `RotX{T}`, `RotY{T}`, `RotZ{T}`
 
     Sparse representations of 3D rotations about the X, Y, or Z axis, respectively.
 
-7. **Two-axis rotations** `RotXY{T}`, etc
+9. **Two-axis rotations** `RotXY{T}`, etc
 
     Conceptually, these are compositions of two of the cardinal axis rotations above,
     so that `RotXY(x, y) == RotX(x) * RotY(y)` (note that the order of application to
     a vector is right-to-left, as-in matrix-matrix-vector multiplication: `RotXY(x, y) * v == RotX(x) * (RotY(y) * v)`).
 
-8. **Euler Angles - Three-axis rotations** `RotXYZ{T}`, `RotXYX{T}`, etc
+10. **Euler Angles - Three-axis rotations** `RotXYZ{T}`, `RotXYX{T}`, etc
 
     A composition of 3 cardinal axis rotations is typically known as a Euler
     angle parameterization of a 3D rotation. The rotations with 3 unique axes,
     such as `RotXYZ`, are said to follow the [**Tait Bryan**](https://en.wikipedia.org/wiki/Euler_angles#Tait.E2.80.93Bryan_angles) angle ordering,
     while those which repeat (e.g. `EulerXYX`) are said to use [**Proper Euler**](https://en.wikipedia.org/wiki/Euler_angles#Conventions) angle ordering.
 
     Like the two-angle versions, the order of application to a vector is right-to-left, so that `RotXYZ(x, y, z) * v == RotX(x) * (RotY(y) * (RotZ(z) * v))`.  This may be interpreted as an "extrinsic" rotation about the Z, Y, and X axes or as an "intrinsic" rotation about the X, Y, and Z axes.  Similarly, `RotZYX(z, y, x)` may be interpreted as an "extrinsic" rotation about the X, Y, and Z axes or an "intrinsic" rotation about the Z, Y, and X axes. 
+    
+### The Rotation Error state and Linearization
+It is often convenient to consider perturbations or errors about a particular 3D rotation, such as applications in state estimation or optimization-based control. Intuitively, we expect these errors to live in three-dimensional space. For globally non-singular parameterizations such as unit quaternions, we need a way to map between the four parameters of the quaternion to this three-dimensional plane tangent to the four-dimensional hypersphere on which quaternions live. 
```

Comment:
Excellent clear discussion on what these maps are for in practice. Thanks!

Actually it turns out I've used error states in the past (I think) while working with @PaulBellette on state estimation for survey aircraft, but I didn't know much about the theory at the time. (Also it was ~ julia 0.3 so we didn't have much tooling!)